### PR TITLE
added title tags for all pages

### PR DESCRIPTION
**Pull request by wilsantow**
Closes issue #3

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ const Home: React.FC<unknown> = () => {
   return (
     <Layout>
       <Head>
-        <title>Electify</title>
-        <link rel="icon" href="/favicon.ico" />
+        <title>Home | Electify</title>
```
Comment (manishprivet) [inline pages/index.tsx:10]: We don't need the Home explicitly on the Homepage, can you change it to just **Electify**

**Review by manishprivet — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (wilsantow) [reply pages/index.tsx:10]: Sure.

**Review by manishprivet — APPROVED** [review]

**Comment by manishprivet** [comment]
Thank you a lot :+1:

**Comment by wilsantow** [comment]
You're welcome

**Pull request merged**